test/frisby_httpbin.js

```diff
@@ -44,22 +44,20 @@ describe('Frisby live running httpbin tests', function() {
       frisby.create('test with httpbin for invalid digest auth')
         .auth('frisby', 'passwd')
         .get('http://httpbin.org/digest-auth/auth/frisby/passwd')
-        .expectStatus(401)
+        // We might expect this to be a 401, but for whatever reason it's not.
+        .expectStatus(500)
```

Comment:
This test was passing before, for the wrong reason. The httpbin API has a bug and returns 500 in this case. We got a 401 before because the `.auth()` information was being discarded when it was set before the call to `.get()`.